src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int top;
+    private final int base;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int top, int base, int height) {
+        super(color);
+        this.top = top;
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (Math.round((float) ((top + base) * height * 10) / 2)) / 10;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + getArea() + " sq. units,"
```

Comment:
Use only English in messages/code. The word 'isosceles' is misspelled as 'isosceless'.